### ISSUE TITLE
Investigate focusOutline inconsistencies in Safari and Firefox

### ISSUE THREAD

**Issue by marcellamaki** [learningequality/kolibri#9070]
## Observed behavior

While doing manual QA for #9038, @radinamatic and I both encountered some inconsistencies with the `focusOutline()` around 

## Expected behavior
`focusOutline()` should be applied (relatively) consistently and surround the entire selected element 

## User-facing consequences
- no focus outline present would be a major blocker for keyboard users navigating in browsers other than Chrome 
- inconsistent outline usage (i.e. not surrounding the same elements across browsers)

## Steps to reproduce
Run 0.15.x local server and navigate by keyboard using the tab key

## Context
So far, this has been replicated on the library page, in the context of the previously mentioned PR, **on macOS and Linux on firefox, and on macOS on Safari.** 

Further investigation of _where_ this is occurring and why it is happening is needed, however.

**Comment by sairina** [comment]
### The following issues have been opened around focus issues in Safari, Firefox, and Chrome:

**Chrome**
1. ["Check" button in exercises](https://github.com/learningequality/kolibri/issues/9181)
2. [App bar - user profile focus ring covers points](https://github.com/learningequality/kolibri/issues/9180)
3. [Folder resource side panel](https://github.com/learningequality/kolibri/issues/9178)
4. [Device permissions `KButton` focus cut off](https://github.com/learningequality/kolibri/issues/9177)
5. [Info panel for videos](https://github.com/learningequality/kolibri/issues/9175)
6. [Videos keyboard navigation not working](https://github.com/learningequality/kolibri/issues/9174)

**Firefox**
1. [Home and library tabs](https://github.com/learningequality/kolibri/issues/9194)
2. [Inconsistency for `FullScreenSidePanel` focus](https://github.com/learningequality/kolibri/issues/9193)
3. [Bookmarks tab content cards](https://github.com/learningequality/kolibri/issues/9192)
4. [Topics page content cards](https://github.com/learningequality/kolibri/issues/9190)
5. [Browsing a channel](https://github.com/learningequality/kolibri/issues/9191)
6. [App bar side navigation](https://github.com/learningequality/kolibri/issues/9189)
7. [App bar/nav bar router links](https://github.com/learningequality/kolibri/issues/9186)
8. [App bar user profile dropdown](https://github.com/learningequality/kolibri/issues/9184)
9. [Device Settings](https://github.com/learningequality/kolibri/issues/9188)
10. [Device Info](https://github.com/learningequality/kolibri/issues/9187)
11. [Sign-in page does not work with keyboard navigation](https://github.com/learningequality/kolibri/issues/9182)

**Safari** (almost completely broken, so only the app bar is noted for now, as fixing this will allow us to investigate further):
1. [App bar navigation](https://github.com/learningequality/kolibri/issues/9183)
